Defs/Ammo/Advanced/12GaugeCharged.xml

```diff
@@ -139,8 +139,6 @@
       </secondaryDamage>
       <armorPenetrationSharp>8</armorPenetrationSharp>      
       <armorPenetrationBlunt>14.4</armorPenetrationBlunt>
-      <pelletCount>6</pelletCount>
```

Comment:
Why revert this change? I was done on purpose to differentiate more from the regular 12 gauge, as seen from #652